partiql-planner/src/main/kotlin/org/partiql/planner/transforms/RexConverter.kt

```diff
@@ -478,20 +480,15 @@ internal object RexConverter {
             val call = when (node.field) {
                 DatetimeField.TIMEZONE_HOUR -> error("Invalid call DATE_DIFF(TIMEZONE_HOUR, ...)")
                 DatetimeField.TIMEZONE_MINUTE -> error("Invalid call DATE_DIFF(TIMEZONE_MINUTE, ...)")
-                else -> call("date_diff_${node.field.name.lowercase()}", arg0, arg1)
+                else -> call("date_diff_${node.field.name.lowercase()}", true, arg0, arg1)
```

Comment:
Since all of these are `call("..", true, ...)` I'm wondering if you should make `call("..", args...)` the default with true. Then you make another function `userCall("..", args...)` or something like that with a better name where the arg is `false`. This would cut down on the diff and we won't have a hanging boolean arg

Reply:
I thought, and internally debated about this. The benefit I see using this approach is that it forces the implementer to explicitly write out the whether or not the function is hidden. 

That said, two functions with different names would work.